src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -489,6 +489,7 @@ public void go() {
           long callValue = TransactionCapsule.getCallValue(trx.getRawData().getContract(0));
           if (callValue > 0) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this code to not nest more than 3 if/for/while/switch/try statements. ([squid:S134](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS134))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2vdPPRVdwyhXLE&open=AWak_C2vdPPRVdwyhXLE)

[ISSUE_KEY]: AWak_C2vdPPRVdwyhXLE